src/from.rs

```diff
@@ -123,6 +123,15 @@ impl fmt::Display for ToFieldError {
 }
 
 impl<const BITS: usize, const LIMBS: usize> Uint<BITS, LIMBS> {
+    pub(crate) const fn const_from_u64(x: u64) -> Self {
```

Comment:
i know this is not for public consumption, but can you document the behavior when the u64 is out of range?